imaging/ml/ml_codelab/breast_density_cloud_ml.ipynb

```diff
@@ -248,7 +249,7 @@
     "%%bash -s {project_id} {jpeg_bucket} {bottleneck_bucket} {validation_percentage} {testing_percentage} {dataflow_num_workers} {staging_bucket}\n",
     "\n",
     "# Install Python library dependencies.\n",
-    "sudo pip3 install google-apitools apache_beam apache_beam[gcp]\n",
+    "sudo pip3 install  tensorflow==1.14.0 google-apitools apache_beam[gcp]==2.18.0 --ignore-installed\n",
```

Comment:
Also why is 1.14 needed?  Or even 1.15. Why cant we use 1.9.x for example? Can you remind me ?

Reply:
in breast_density_cloud_ml.ipynb  is installed  as dependency and not actually used, it may be any version if it have no conflicts with over packages, but i've specified 1.14.0 to be same as in breast_density_auto_ml.ipynb

Reply:
So in auto_ml why 1.14?

Reply:
hm, originally it was to be corresponded with dataflow, i'll test with 1.15 and update this comment

Reply:
yep, it seems to work fine

Reply:
Great can you please update.

Reply:
I think you forgot to update here!

Reply:
>I think you forgot to update here!

no, just jupyterlab not quite friendly environment for developer:)